ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -36,21 +33,10 @@ def _trunk_groups_mlag_l3_name(self) -> str:
     def _trunk_groups_uplink_name(self) -> str:
         return get(self.shared_utils.trunk_groups, "uplink.name", required=True)
 
-    @cached_property
```

Comment:
Why are  you removing these?

Reply:
Notice it is "dangerous" to move things with "switch.*" around. Since the content might be different depending on when it is executed/read.

Reply:
not used anywhere